Marlin/Configuration.h

```diff
@@ -2274,8 +2274,8 @@
 #if EITHER(AUTO_BED_LEVELING_LINEAR, AUTO_BED_LEVELING_BILINEAR)
 
   // Set the number of grid points per dimension.
-  #define GRID_MAX_POINTS_X 5
-  #define GRID_MAX_POINTS_Y GRID_MAX_POINTS_X
+  #define GRID_MAX_POINTS_X 7
```

Comment:
not suitable for other printers with rectangular bed.

Reply:
should fit for 4MaxPro1 and 4MaxPro2